src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -3531,6 +3531,11 @@ public override SyntaxNode ExpressionStatement(SyntaxNode expression)
             return SyntaxFactory.ExpressionStatement((ExpressionSyntax)expression);
         }
 
+        public override SyntaxNode ConditionalAccessExpression(SyntaxNode expression, SyntaxNode whenNotNull)
+        {
+            return SyntaxFactory.ConditionalAccessExpression(Parenthesize(expression), SyntaxFactory.MemberBindingExpression((SimpleNameSyntax)whenNotNull));
```

Comment:
Also, now that I am reusing the existing method `MemberAccessExpression` for extra tests, the built expression gets invalid parenthesis, such as:
```csharp
VerifySyntax<ConditionalAccessExpressionSyntax>(_g.ConditionalAccessExpression(_g.IdentifierName("x"), _g.MemberAccessExpression(_g.MemberBindingExpression(_g.IdentifierName("y")), _g.IdentifierName("z"))), "(x)?(.y).z");
```

where `(x)?(.y).z` is not legal C#. Am I missing something?